os-sso71/module.yaml

```diff
@@ -5,7 +5,18 @@ description: Legacy os-sso71 script package.
 
 modules:
   install:
+  - name: os-eap-launch
   - name: os-eap-migration
 
 execute:
 - script: configure.sh
+
+run:
+      user: 185
+      cmd:
+          - "/opt/eap/bin/openshift-launch.sh"
+
+envs:
+    - name: "CLI_GRACEFUL_SHUTDOWN"
```

Comment:
Do we want to define this in os-eap-launch and remove it from the other modules?

Reply:
Jonathan's change above introduced the ```os-eap-launch``` module. But per code review (```runServer``` routine IIRC), the ```CLI_GRACEFUL_SHUTDOWN``` variable is supported only by EAP 7.x (and derived) images, thus used ```os-eap7-launch``` module instead (and remove the duped variable occurrences).